Mage.Sets/src/mage/cards/b/BloodflowConnoisseur.java

```diff
@@ -31,7 +31,7 @@ public BloodflowConnoisseur(UUID ownerId, CardSetInfo setInfo) {
         this.toughness = new MageInt(1);
 
         // Sacrifice a creature: Put a +1/+1 counter on Bloodflow Connoisseur.
-        Ability ability = new SimpleActivatedAbility(Zone.BATTLEFIELD, new AddCountersSourceEffect(CounterType.P1P1.createInstance()), new ManaCostsImpl());
+        Ability ability = new SimpleActivatedAbility(Zone.BATTLEFIELD, new AddCountersSourceEffect(CounterType.P1P1.createInstance()), new ManaCostsImpl<>());
         ability.addCost(new SacrificeTargetCost(new TargetControlledCreaturePermanent(FILTER_CONTROLLED_CREATURE_SHORT_TEXT)));
```

Comment:
This seems weird. Why is an empty ManaCostImpl being used only to then add a SacrificeTargetCost?

Reply:
Fixed, thanks for pointing out. imo for big changes like this it's nice to restrict them to a single purpose (in this case, adding the diamond operator to usages of `ManaCostsImpl` to avoid raw usage or this parameterized class). That being said, agreed that BloodflowConnoisseur's syntax could be improved.

Reply:
I agree with you, this just happened to be both a) a directly related to the refactoring you were doing, and b) a small fix.